release/rotation

```diff
@@ -1,3 +1,4 @@
+U021DSF26BS victormueller-da
```

Comment:
let’s give him a week or even start him at the bottom :slightly_smiling_face:

Reply:
@realvictorprm what do you think? Do you want to do the release this week (Wednesday), or would you rather have a bit more time? Either way, I'd be happy to walk you through the release process on Zoom for your first time.